packages/mds-agency/tests/integration-tests.spec.ts

```diff
@@ -705,7 +710,8 @@ describe('Tests API', () => {
       .post(pathPrefix(`/vehicles/${TRIP_UUID}/event`))
       .set('Authorization', AUTH)
       .send({
-        event_type: VEHICLE_EVENTS.maintenance_pick_up,
+        event_types: [VEHICLE_EVENTS.maintenance_pick_up],
```

Comment:
Can we change `VEHICLE_EVENTS` and other enums to use `const` assertion like all of the newer type do:
```ts
const VEHICLE_EVENTS = ['maintenance_pick_up', ...] as const
type VEHICLE_EVENT = typeof VEHICLE_EVENTS[number]
```

Reply:
Yep, after merging this that's next up on my list of things to do to prep for Taxi.

Reply:
Merged in a PR which makes this switch